src/box/session.cc

```diff
@@ -375,3 +377,71 @@ generic_session_sync(struct session *session)
 	(void) session;
 	return 0;
 }
+
+API_EXPORT int
+box_session_id()
+{
+	return current_session()->id;
+}
+
+API_EXPORT bool
+box_session_exists(uint64_t sid)
+{
+	return (session_find(sid) != NULL);
+}
+
+API_EXPORT const char *
+box_session_peer(uint64_t sid)
+{
+	struct session *session = session_find(sid);
+	if (session == NULL)
+		session = current_session();
```

Comment:
That looks wrong. If I would pass any random number not matching a session, the last thing I would expect is to get my own peer. It is also inconsistent with the Lua API, which throws an error if couldn't find the session.